docs/articles/rag-application-communication-system.md

```diff
@@ -0,0 +1,212 @@
+
+Our RAG projects at PleIAs were conceived in the context of high constraints for verifiability (public sector, large legacy privacy companies). We took inspiration from the collaborative encyclopedia, Wikipedia, who faced constraints similar to those of an expert RAG system: texts cannot be attributed to individual authors who take responsibility for their publication and trustworthiness. Accuracy stems mostly from verifiability, by associating individual statements with secondary sources and easing verification [easing verification by associating individual statements with secondary sources + exact passage when they appear, to limit supplementary work of cross-checking].
+
+While it's possible to get to the same output using a powerful model (and this was actually the source for the instruction dataset)
+[the first iteration of the model was trained for the French public services and is available on HuggingFace: https://huggingface.co/AgentPublic/guillaumetell-7b We're working right now on a better version (also fully multilingual).
+Otherwise: "while it is possible to attain similar results with Frontier LLMs without fine-tuning, this has some limitations".]
+, this has some limitations:
+
+* **Prompts won't work systematically**. While generating the instruction dataset, I’ve had to discard 5% of the generated output. This is not much, but when you combine this uncertainty with all the others (retrieval failure, imprecise query) and this adds up to the stress of your RAG system. [While generating instruction datasets for RAG, I usually discard a non-negligible share of generated output (5-10%).]
+* **Prompts can saturate**, especially if on top of asking for an advanced verifiability check, you are also making other recommendations for style (see section 4.3 below).
+* **Prompts are vulnerable to injection and security issues**. This is the infamous “Chevrolet” effect where a help support system turns out to be a thin wrapper of ChatGPT.
+
+**Assessing model robustness is tricky**. Standard benchmarks don't apply; because specialization through retraining results only in a general performance loss, this is what you are measuring.
+
+The most straightforward way of measuring accuracy is to compare the model output and the original documents. This comparison obviously works best if you're already clarifying the attribution of individual statements to specific documents through generated references. For this process you can use LLM-as-a-judge, or, if you provide actual quotes, a straight (?) comparison with the original text (as an additional confusing factor, this is called a “text alignment” algorithm, but has nothing to do with fine-tuning or RLHF).
+
```

Comment:
is this "audience alignment gap" different from a "user alignment gap"?

Reply:
no. "user alignment gap" is better in fact in this context.